contrib/deploy-argocd/01-namespace.yml

```diff
@@ -0,0 +1,4 @@
+kind: Namespace
```

Comment:
Why do we need argocd? Maybe I'm wrong or missing something but I think we don't need argocd to deploy it